### PR TITLE
Remove Android bits and add limitations on ios platform views

### PR DESCRIPTION
**Pull request by leighajarett**
Moving from previous PR: https://github.com/flutter/website/pull/8793

Fixes https://github.com/flutter/website/issues/8741

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/platform-integration/ios/platform-views.md [hunk 1]
```diff
@@ -355,7 +355,41 @@ Widget build(BuildContext context) {
...
+- The [`ShaderMask`][] and [`ColorFiltered`][] widgets are not supported.
+- The [`BackdropFilter`][] widget is supported, 
+but there are some limitations on how it can be used. 
+See the [iOS Platform View Backdrop Filter Blur design doc] for details.
+
+[`ShaderMask`]: {{site.api}}/flutter/foundation/ShaderMask.html
+[`ColorFiltered`]: {{site.api}}/flutter/foundation/ColorFiltered.html
+[`BackdropFilter`]: {{site.api}}/flutter/foundation/BackdropFilter.html
+[`defaultTargetPlatform`]: {{site.api}}/flutter/foundation/defaultTargetPlatform.html
+[iOS Platform View Backdrop Filter Blur design doc]: flutter.dev/go/ios-platformview-backdrop-filter-blur
```
Comment (sfshaza2) [inline src/platform-integration/ios/platform-views.md:394]:
```suggestion
[design-doc]: {{site.main-url}}/go/ios-platformview-backdrop-filter-blur
```

File: src/platform-integration/ios/platform-views.md [hunk 2]
```diff
@@ -355,7 +355,41 @@ Widget build(BuildContext context) {
...
+In other words, if an animation is slow while a platform view is rendered, 
+then consider taking a screenshot of the native view and rendering it as a texture.
+
+## Composition limitations
+There are some limitations when composing iOS Platform Views.
+
+- The [`ShaderMask`][] and [`ColorFiltered`][] widgets are not supported.
+- The [`BackdropFilter`][] widget is supported, 
+but there are some limitations on how it can be used. 
+See the [iOS Platform View Backdrop Filter Blur design doc] for details.
```
Comment (sfshaza2) [inline src/platform-integration/ios/platform-views.md:388]:
```suggestion
For more details, check out the [iOS Platform View Backdrop Filter Blur design doc][design-doc].
```

As per GSDG, the condition goes first.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
LGTM, after fixing a couple links. Also, even though you only modified the _comment_ in a Dart code snippet, the build is failing because you need to run the tool/refreshcodeexcerpts command.

**Comment by leighajarett** [comment]
@atsansone can you approve this?

File: src/platform-integration/ios/platform-views.md [hunk 3]
```diff
@@ -192,8 +192,8 @@ class FLPlugin: NSObject, FlutterPlugin {
 
 {% sample Objective-C %}
 
-Add the headers for the factory and the platform view.
-For example, `FLNativeView.h`:
+In Objective-C, add the headers for the factory and the platform view.
+For example, as shown in `FLNativeView.swift`:
```
Comment (cyanglaz) [inline src/platform-integration/ios/platform-views.md:196]: Should this be `FLNativeView.h` for Objective-C?

**Review by sfshaza2 — APPROVED** [review]
lgtm, once @cyanglaz's question is addressed!

**Review by atsansone — APPROVED** [review]

**Pull request merged**